migrations/V010__drop_test_data_and_add_official_seasons.sql

```diff
@@ -0,0 +1,158 @@
+DELETE FROM league_season_score;
+DELETE FROM league_season_division_subdivision;
+DELETE FROM league_season_division;
+DELETE FROM league_season;
+DELETE FROM league;
+
+INSERT INTO leaderboard (id, technical_name) VALUES
+  (4, "tmm_4v4_full_share"),
+  (5, "tmm_4v4_share_until_death");
+
+INSERT INTO league (id, technical_name, name_key, description_key, image_url, medium_image_url, small_image_url) VALUES
+  (1, "1v1_league", "1v1_league.name", "1v1_league.description",
+  "https://content.faforever.com/divisions/icons/", "https://content.faforever.com/divisions/icons/medium/", "https://content.faforever.com/divisions/icons/small/"),
+  (2, "2v2_league", "2v2_league.name", "2v2_league.description",
+  "https://content.faforever.com/divisions/icons/", "https://content.faforever.com/divisions/icons/medium/", "https://content.faforever.com/divisions/icons/small/"),
+  (3, "4v4_full_share_league", "4v4_full_share_league.name", "4v4_full_share_league.description",
```

Comment:
Probably makes sense to keep the name keys consistent with the ones from the other tables
```suggestion
  (3, "4v4_full_share_league", "4v4_fs_league.name", "4v4_fs_league.description",
```

Reply:
I wanted to have it in full somewhere so one can get an idea what fs and sud means, but I guess the leaderboard could be sufficient